examples/experimental/i3s-17-and-debug/components/tile-validator.js

```diff
@@ -235,6 +238,20 @@ export default class TileValidator extends PureComponent {
     };
   }
 
+  getNormalControlsStyle(isTileHasNormals, element = null) {
```

Comment:
```suggestion
  getNormalControlsStyle(hasNormals, element = null) {
```

Reply:
@ibgreen @belom88 yes it would be good to show which attributes are present in the debugger window. Yes it would be good to generate face normals (maybe offer it thru a checkbox when its not present) when its absent/missing (arcgis clients do the same).